doc/Guides/DevelopmentServer.md

```diff
@@ -0,0 +1,239 @@
+# Development Servers
+
+During this guide we will set up a dedicated server and place the server build of our project on the dedicated server.
+
+**IMPORTANT: Service providers are NOT free. Take note that Mirror is not affiliated and can not be held responsible for any charges or
```

Comment:
Maybe we can make this into a bulleted list, with a bold heading that "Before we begin, there are some potential problems you may face, as no server is truly 'free'"?